### PR TITLE
Add build and deploy test

### PR DESCRIPTION
**Pull request by mateoflorido**
## Changes
- Create Build and deploy integration test

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+venv/
+build/
+*.charm
+.tox/
+htmlcov/
+.coverage
+__pycache__/
+*.py[cod]
+juju-crashdump-*
+regsync
```
Comment (addyess) [inline .gitignore:10]: i'm not sure `regsync` is in use in this project :)

**Review by addyess — APPROVED** [review]
LGTM

File: .jujuignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+*.charm
+.tox/
+htmlcov/
+.coverage
+regsync
```
Comment (addyess) [inline .jujuignore:5]: same

**Comment by Cynerva** [comment]
I've opened https://github.com/charmed-kubernetes/charm-kube-ovn/issues/7 about the failing `kubectl apply` stacktraces in the test output. It's a known issue that exists prior to this PR, but one that would be nice to address later.

**Review by Cynerva — APPROVED** [review]
This looks great. Thanks!

**Pull request merged**